### ISSUE TITLE
Local Build Issue

### ISSUE THREAD

**Issue by cbserra** [PedalPlayground/PedalPlayground.github.io#868]
I'm on an M1 MacBook Pro, and run the commands outlined in the README in a Rosetta Terminal (so I have side-by-side x86 and ARM Brew installations). 

Here's my output when running `gulp`. I'm running this off of my fork:

`❯ gulp
[21:22:30] Using gulpfile ~/Dev/git/PedalPlayground.github.io.cbserra/gulpfile.js
[21:22:30] Starting 'styles'...
[21:22:30] Starting 'scripts'...
[21:22:30] Starting 'watch'...
[21:22:30] Finished 'watch' after 15 ms
[21:22:30] gulp-notify: [Task Failed [gulp-sass]] See console.
TASK: [gulp-sass]
PROB: bower_components/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss
Error: Invalid CSS after "...     floor(math": expected expression (e.g. 1px, bold), was ".div($grid-gutter-w"
        on line 369 of bower_components/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss
>> $navbar-padding-horizontal:        floor(math.div($grid-gutter-width, 2)) !def
   -----------------------------------------^

[21:22:30] Finished 'styles' after 95 ms
[21:22:32] gulp-notify: [Gulp notification] JS Compiled!
[21:22:32] /Users/cbserra/Dev/git/PedalPlayground.github.io.cbserra/public/scripts/scripts-min.js.gz reloaded.
[21:22:32] gulp-notify: [Gulp notification] JS Compiled!
[21:22:32] /Users/cbserra/Dev/git/PedalPlayground.github.io.cbserra/public/scripts/scripts.js.gz reloaded.
[21:22:32] Finished 'scripts' after 1.93 s
[21:22:32] Starting 'default'...
[21:22:32] Finished 'default' after 72 μs
^C
`

First, I get that `gulp-sass` error. Even after running `sudo npm rebuild node-sass -g`, the error is present.

Also, the script never completes. I have to Ctrl+C to exit.

**Comment by tehtrav** [comment]
Yeah, my apologies. I keep meaning to rework the project dependencies but haven't had a chance to do so yet.

**Comment by cbserra** [comment]
Ok, `gulp` not completing makes sense, once I learned about `"watch"` -- that process stays open so when local changes are made it will re-deploy. so that's a process to leave up in a separate terminal window.

I did downgrade `"bootstrap-sass": "^3.3.7",` to `"3.4.1"` (I imaging it was updating to < version `4.x.x` based on the `^`.

Now, I see `bower.json` specifies jQuery 3.1.1, but the scripts deployed to `public/` are `3.4.1` -- is there a reason for that?
